internal/handler/user.go

```diff
@@ -2,20 +2,34 @@ package handler
 
 import (
 	"context"
-	"net/http"
-
 	"github.com/dilyara4949/flight-booking-api/internal/domain"
 	"github.com/dilyara4949/flight-booking-api/internal/handler/request"
 	"github.com/dilyara4949/flight-booking-api/internal/handler/response"
 	"github.com/gin-gonic/gin"
 	"github.com/google/uuid"
+	"net/http"
 )
 
 type UserService interface {
 	CreateUser(ctx context.Context, signup request.Signup, password string) (domain.User, error)
+	ResetPassword(ctx context.Context, req request.ResetPassword, requirePasswordReset bool) error
 	DeleteUser(ctx context.Context, id uuid.UUID) error
 	ValidateUser(ctx context.Context, signin request.Signin) (domain.User, error)
-	ResetPassword(ctx context.Context, req request.ResetPassword, requirePasswordReset bool) error
+	GetAll(ctx context.Context, page, pageSize int) ([]domain.User, error)
```

Comment:
I would suggest renaming to "GetUsers", "GetList" or similar, since we applying pagination here